gm4_ambrosia/data/ambrosia/functions/honey_tanks/liquid_init_honey.mcfunction

```diff
@@ -0,0 +1,5 @@
+data merge block ~ ~ ~ {CustomName:'{"translate":"%1$s","with":["Honey Tank","TRANSPATION.KEY.FOR.MISODE"]}'}
```

Comment:
`TRANSPATION` lol. Needs to be filled in

Reply:
`block.gm4.liquid_tank.honey`